string port
---
bool status
string msg